src/wasm2js.h

```diff
@@ -2278,7 +2279,7 @@ void Wasm2JSGlue::emitMemory(
 
   for (auto& seg : wasm.memory.segments) {
     assert(!seg.isPassive && "passive segments not implemented yet");
-    out << segmentWriter << "(" << globalOffset(seg) << ", \""
+    out << "base64DecodeToExistingUint8Array(new Uint8Array(" << buffer << "), " << globalOffset(seg) << ", \""
```

Comment:
this doesn't look right - for each segment it will do `new Uint8Array`..?

Reply:
Yeah have to create a view to the buffer. :/ I’ll see about sharing the view when I have a moment.